client/my-sites/plans/jetpack-plans/style.scss

```diff
@@ -1,11 +1,65 @@
+/* cyrillic-ext */
 @font-face {
-	font-display: swap;
-	font-family: 'Noto Sans SemiCondensed Bold';
-	font-weight: 700;
-	src:
-		url( 'https://s1.wp.com/i/fonts/notosans/NotoSans-SemiCondensedBold.woff2' ) format( 'woff2' ),
-		url( 'https://s1.wp.com/i/fonts/notosans/NotoSans-SemiCondensedBold.woff' ) format( 'woff' ),
-		url( 'https://s1.wp.com/i/fonts/notosans/NotoSans-SemiCondensedBold.ttf' ) format( 'truetype' );
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvvYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
+}
+/* cyrillic */
+@font-face {
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvmYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
+}
+/* greek-ext */
+@font-face {
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvuYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+1F00-1FFF;
+}
+/* greek */
+@font-face {
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvhYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+0370-03FF;
+}
+/* vietnamese */
+@font-face {
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvtYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB;
+}
+/* latin-ext */
+@font-face {
+  font-family: 'Inter';
+  font-style: oblique 0deg 10deg;
+  font-weight: 400 700;
+  font-display: swap;
+  src: url( 'https://fonts.gstatic.com/s/inter/v7/UcCo3FwrK3iLTcvsYwYZ8UA3J58.woff2' ) format( 'woff2' );
+  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
```

Comment:
Out of curiosity, how did you generate these font-face definitions?

Reply:
Sorry for the delay, @monsieur-z — I grabbed those directly from Google's CSS file that was generated for this diff on Jetpack.com: D70711-code

https://fonts.googleapis.com/css2?family=Inter:slnt,wght@-10..0,400..700&display=swap

Not sure if we should be more embracing of other sets of languages or less so to save on bandwidth but I'm posting about this work and will bring this up.

Reply:
Nice! Given it was generated by Google Fonts, that's likely the exhaustive set of writing systems supported by that font. Browsers seem to only load the font file that matches the user's system, so I don't think it's necessary to reduce this set.